mk/scripts/db-magerun.sh

```diff
@@ -14,13 +14,14 @@ done
 
 # Use MAGERUN_STRIP value, if it does not exist use default values
 MAGERUN_STRIP="${MAGERUN_STRIP:-@2fa @aggregated @customers @idx @log @oauth @quotes @replica @sales @search @sessions @stripped @trade @temp}"
+MAGERUN_EXCLUDE=""
```

Comment:
Doesn't this value need to be grabbed from the user input when using the command? I think it'll always be `""` right now.